app/api/v1/endpoints/user_actions.py

```diff
@@ -0,0 +1,19 @@
+from flask import request
+from flask_restplus import Resource
+from app.api.v1.api import api
+from app.api.v1.models.user_actions import user_action
+from app.user_activity_gatherer import UserActivityGatherer
+
+user_actions_name_space = api.namespace("user_actions", "Endpoint used for sending user actions")
+
+
+@user_actions_name_space.route("")
+class UserAction(Resource):
+    """Allows to send user actions"""
+    @api.expect(user_action, validate=True)
+    @api.response(201, "User action successfully sent")
+    def post(self):
+        user_action = request.get_json()
+        UserActivityGatherer.save_user_action(user_action)
+
+        return "OK", 201
```

Comment:
If there is no 'usefull' information (like object being created) then no message content should be returned (simple status code is enough).

Additionally - if you use `201` return code then you should adhere to the standard:

```
The new resource is effectively created before this response is sent back and the new resource is returned in the body of the message
```

You can choose either generic `200` or maybe `204 No Content`, which might be a good fit here.

Reply:
fixed